### PR TITLE
Ajay - Added Compound Button, Conditional Action Button, and Scrollab…

### PR DESCRIPTION
**Pull request by Harikrishnan-MSFT**
…le Conatiner in Adaptive Cards (Csharp, Nodejs)

### CODE REVIEW

File: samples/bot-formatting-cards/nodejs/bots/botFormattingCards.js [hunk 1]
```diff
@@ -213,6 +253,50 @@ class BotFormattingCards extends ActivityHandler {
...
+
+    /**
+     Sends a Compound Button adaptive card
+    */
+     SendCompoundButtonCard() {
+        return CardFactory.adaptiveCard(adaptiveCardCompoundButton);
+    }
+
     /**
    * Send AdaptiveCard Fromats to the user.
```
Comment (Pawank-MSFT) [inline samples/bot-formatting-cards/nodejs/bots/botFormattingCards.js:301]: please correct fromat to format

**Review by Pawank-MSFT — COMMENTED** [review]
please fix small spell mistakes.

**Review by dclaux — APPROVED** [review]

File: samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs [hunk 2]
```diff
@@ -239,5 +247,120 @@ public static Attachment SendRoundedCornerCard()
...
+            {
+                ContentType = contentType,
+                Content = JsonConvert.DeserializeObject(adaptiveCardConditionalJson),
+            };
+
+            return ConditionalCardAttachment;
+        }
+
+        /// <summary>
+        /// Scorrable container for adaptive Card
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs]: Fixed

File: samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs [hunk 3]
```diff
@@ -239,5 +247,120 @@ public static Attachment SendRoundedCornerCard()
...
+            {
+                ContentType = contentType,
+                Content = JsonConvert.DeserializeObject(adaptiveCardStarRatingsJson),
+            };
+
+            return StarRatingsCardAttachment;
+        }
+
+        /// <summary>
+        /// Conditinal enablement of action button
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/bot-formatting-cards/csharp/BotFormattingCards/Cards/AllCards.cs]: Fixed

File: samples/bot-formatting-cards/nodejs/Readme.md [hunk 4]
```diff
@@ -247,10 +247,80 @@ The simplest way to run this sample in Teams is to use Teams Toolkit for Visual
...
+
+**Star ratings in Adaptive Cards validation:**
+
+![StarRatingsAdaptiveCardsValidation](Images/27.StarRatingsAdaptiveCardsValidation.png)
+
+**Star Ratings Feedback:**
+
+![StarRatingsFeedback](Images/28.StarRatingsFeedback.png)
+
+**Conditinal Action and Scrollable Container Buttons:**
```
[the remark this answers is not in the document]
Comment (Pawank-MSFT) [reply samples/bot-formatting-cards/nodejs/Readme.md]: Fixed

File: samples/bot-formatting-cards/nodejs/bots/botFormattingCards.js (hunk 1 above)
Comment (Pawank-MSFT) [reply samples/bot-formatting-cards/nodejs/bots/botFormattingCards.js]: Fixed

**Review by Pawank-MSFT — APPROVED** [review]
Looks correct, Approving!

**Review by Mohammed-MSFT — APPROVED** [review]

**Review by Wajeed-msft — APPROVED** [review]

**Comment by Harikrishnan-MSFT** [comment]
Build Pipeline - 
https://github.com/OfficeDev/Microsoft-Teams-Samples/actions/runs/11697865086

**Pull request merged**